{"uid":"2217e2599cad2aa1","name":"Invalid parameters handling -- @1.14 ","fullName":"Hoover service testing: Invalid parameters handling","historyId":"f01246779be7a7c9072398ff8bbdfe24","time":{"start":1682669937094,"stop":1682669937100,"duration":6},"status":"failed","statusMessage":"AssertionError: Expected 400 but got 200\n","statusTrace":"  File \"/home/luis/.local/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/luis/.local/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/hoover_nonfunctional_steps.py\", line 18, in step_impl_then_service_should_return_bad_request_error\n    assert context.response_status_code == 400, f\"Expected 400 but got {context.response_status_code}\"\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Expected 400 but got 200\n","statusTrace":"  File \"/home/luis/.local/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/luis/.local/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/hoover_nonfunctional_steps.py\", line 18, in step_impl_then_service_should_return_bad_request_error\n    assert context.response_status_code == 400, f\"Expected 400 but got {context.response_status_code}\"\n","steps":[{"name":"Given a request with wrong parameters \"5\" \"5\" \"6\" \"7\" \"1,1\" \"N\"","time":{"start":1682669937095,"stop":1682669937095,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"When I send a request with wrong parameters","time":{"start":1682669937095,"stop":1682669937099,"duration":4},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Then the service should return a bad request error","time":{"start":1682669937099,"stop":1682669937100,"duration":1},"status":"failed","statusMessage":"AssertionError: Expected 400 but got 200\n","statusTrace":"  File \"/home/luis/.local/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/luis/.local/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/hoover_nonfunctional_steps.py\", line 18, in step_impl_then_service_should_return_bad_request_error\n    assert context.response_status_code == 400, f\"Expected 400 but got {context.response_status_code}\"\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Hoover service testing"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"dirt_positions","value":"1,1"},{"name":"instructions","value":"N"},{"name":"room_x","value":"5"},{"name":"room_y","value":"5"},{"name":"start_x","value":"6"},{"name":"start_y","value":"7"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"2217e2599cad2aa1.json","parameterValues":["1,1","N","5","5","6","7"]}